content/docs/start/data-pipelines.md

```diff
@@ -260,22 +259,22 @@ Stage 'prepare' didn't change, skipping
 Stage 'featurize' didn't change, skipping
 ```
 
-Same as before, no need to run `prepare`, `featurize`, etc ... but, it doesn't
-run even `train` again this time either! It cached the previous run with the
-same set of inputs (parameters + data) and reused it.
+As before, there was no need to rerun `prepare`, `featurize`, etc. But this time
+it also doesn't rerun `train`! The previous run with the same set of inputs
+(parameters & data) was saved in DVC's <abbr>run-cache</abbr>, and reused here.
 
 </details>
 
 <details>
 
 ### 💡 Expand to see what happens under the hood.
 
-`dvc repro` relies on the DAG definition that it reads from `dvc.yaml`, and uses
-`dvc.lock` to determine what exactly needs to be run.
+`dvc repro` relies on the <abbr>DAG</abbr> definition which it reads from
+`dvc.yaml`, and uses `dvc.lock` to determine what exactly needs to be run.
 
-`dvc.lock` file is similar to `.dvc` files and captures hashes (in most cases
-`md5`s) of the dependencies, values of the parameters that were used, it can be
-considered a _state_ of the pipeline:
+Meanwhile, the `dvc.lock` file is similar to a `.dvc` file — it captures hashes
+(in most cases `md5`s) of the dependencies and values of the parameters which
+were used. It can be considered a _state_ of the pipeline:
```

Comment:
Just removing "Meanwhile" since `dvc.lock` was already mentioned in the last p. Also, this that was needed I think (same reasons for few other instanced I rolled back).
```suggestion
The `dvc.lock` file is similar to a `.dvc` file — it captures hashes (in most
cases `md5`s) of the dependencies and values of the parameters that were used.
It can be considered a _state_ of the pipeline:
```